.gitignore

```
@@ -104,4 +104,7 @@ typings/
 package-lock.json
 
 # translations are stored in the `i18n` via crowdin
-i18n
\ No newline at end of file
+i18n
+
+# code-import
+code/node_modules
```

let's also ignore all the lock files in here:
```suggestion
code/node_modules
code/package-lock.json
code/yarn.lock
code/pnpm-lock.yaml
```